client/src/global.css

```diff
@@ -483,7 +498,7 @@ TODO: override default styles
 }
 
 .rr-response {
-  background-color: rgb(229, 229, 229);
+  background-color: calc(var(--background-color) + 2%);
```

Comment:
Is this increasing the hue of the color? Didn't know you could do this hex colors.

Reply:
probably cant... othewrwise i will need to create another color var